dojo/api_v2/serializers.py

```diff
@@ -388,6 +390,65 @@ class BurpRawRequestResponseSerializer(serializers.Serializer):
     req_resp = RequestResponseSerializerField(required=True)
 
 
+class BurpRawRequestResponseMultiSerializer(serializers.ModelSerializer):
+    burpRequestBase64 = serializers.CharField()
+    burpResponseBase64 = serializers.CharField()
+
+    def get_stringrequest(self, obj) -> str:
+        result = ""
+        if isinstance(obj, dict):
+            result = obj.get("string_request", "")
+        elif obj.string_request:
+            result = obj.string_request
+
+        return result
+
+    def get_stringresponse(self, obj) -> str:
+        result = ""
+        if isinstance(obj, dict):
+            result = obj.get("string_response", "")
+        elif obj.string_response:
+            result = obj.string_response
+
+        return result
+
+    def to_representation(self, data):
+        return {
+            "id": data.id,
+            "finding": data.finding.id,
+            "burpRequestBase64": data.burpRequestBase64.decode("utf-8"),
+            "burpResponseBase64": data.burpResponseBase64.decode("utf-8"),
+        }
+
+    def validate(self, data):
+        b64request = data.get("burpRequestBase64", None)
+        b64response = data.get("burpResponseBase64", None)
+        finding = data.get("finding", None)
+
+        if not b64request or not b64response or not finding:
+            msg = "burpRequestBase64, burpResponseBase64, and finding are required."
+            raise ValidationError(msg)
+
+        base64.b64decode(b64request, validate=True)
+        base64.b64decode(b64response, validate=True)
```

Comment:
Might want to wrap these in a `try`/`except` so that we can raise a `ValidationError` instead of a 500 if base64 decoding fails.

**Request:**
```
❯ curl -XPOST -H "Authorization: ${DD_AUTH}" -H "Content-Type: application/json" http://localhost:8080/api/v2/request_response_pairs/ -d '{"finding": 908, "burpRequestBase64": "hi", "burpResponseBase64": "hi"}'
{"message":"Internal server error, check logs for details"}
```

**Logs:**
```
uwsgi-1             | [11/Dec/2024 01:14:31] ERROR [dojo.api_v2.exception_handler:48] Incorrect padding
uwsgi-1             | Traceback (most recent call last):
uwsgi-1             |   File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 506, in dispatch
uwsgi-1             |     response = handler(request, *args, **kwargs)
uwsgi-1             |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
uwsgi-1             |   File "/usr/local/lib/python3.11/site-packages/rest_framework/mixins.py", line 18, in create
uwsgi-1             |     serializer.is_valid(raise_exception=True)
uwsgi-1             |   File "/usr/local/lib/python3.11/site-packages/rest_framework/serializers.py", line 223, in is_valid
uwsgi-1             |     self._validated_data = self.run_validation(self.initial_data)
uwsgi-1             |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
uwsgi-1             |   File "/usr/local/lib/python3.11/site-packages/rest_framework/serializers.py", line 445, in run_validation
uwsgi-1             |     value = self.validate(value)
uwsgi-1             |             ^^^^^^^^^^^^^^^^^^^^
uwsgi-1             |   File "/app/dojo/api_v2/serializers.py", line 432, in validate
uwsgi-1             |     base64.b64decode(b64request, validate=True)
uwsgi-1             |   File "/usr/local/lib/python3.11/base64.py", line 88, in b64decode
uwsgi-1             |     return binascii.a2b_base64(s, strict_mode=validate)
uwsgi-1             |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
uwsgi-1             | binascii.Error: Incorrect padding
uwsgi-1             | [11/Dec/2024 01:14:31] ERROR [django.request:248] Internal Server Error: /api/v2/request_response_pairs/
```